NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/player/SurvivalFly.java

```diff
@@ -1695,6 +1684,9 @@ else if (Bridge1_13.isRiptiding(player) || (data.timeRiptiding + 3000 > now)) {
                 else if ((totalVDistViolation < 0.8 && data.liftOffEnvelope == LiftOffEnvelope.LIMIT_LIQUID)) {
                     // Ignore water logged blocks 
                 }
+                else if (Bridge1_9.isGlidingWithElytra(player)){
+                    // Don't care about players gliding
+                }
```

Comment:
It won't work as the method returns true, they would stay on CreativeFly instead. If it was that simple, I wouldn't have to add tons of fake velocities thing.

Reply:
Yeah i just thought about it. I think your fix with data#keepFrictionTick fixed it anyway, will remove this one.